### PR TITLE
examples: Fix more deprecations/warnings in configs

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message: examples: Fix more deprecations/warnings in configs
Additional Description:

further to #13503 

Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

**Comment by phlax** [comment]
doing `git grep v2 examples/` i think i have caught most of the `v2` deprecated configs - there are some remaining references, particularly in the go implementations

File: examples/jaeger-tracing/front-envoy-jaeger.yaml [hunk 1]
```diff
@@ -15,9 +15,9 @@ static_resources:
             provider:
               name: envoy.tracers.zipkin
               typed_config:
-                "@type": type.googleapis.com/envoy.config.trace.v2.ZipkinConfig
+                "@type": type.googleapis.com/envoy.config.trace.v3.ZipkinConfig
                 collector_cluster: jaeger
-                collector_endpoint: "/api/v2/spans"
+                collector_endpoint: "/api/v3/spans"
```
Comment (mattklein123) [inline examples/jaeger-tracing/front-envoy-jaeger.yaml:20]: I'm guessing this change needs to be reverted. I think this is a Zipkin thing? Same elsewhere?

**Review by mattklein123 — CHANGES_REQUESTED** [review]
Thanks (though again I would really like to understand why this is not failing as I thought we fail config checking when using a deprecated field).

/wait

**Comment by phlax** [comment]
@mattklein123 im afraid that is beyond my immediate knowledge of how the config validation works

Shall i open a separate ticket, and then we can land this ?

**Comment by mattklein123** [comment]
> Shall i open a separate ticket, and then we can land this ?

Yes that's fine. I think this may be specific to the deprecated typed_config that @htuch recently worked on, but I'm not sure.

**Comment by htuch** [comment]
Let's discuss in the other thread.

**Comment by phlax** [comment]
> Let's discuss in the other thread.

i pushed the thread to #13542

File: examples/jaeger-tracing/front-envoy-jaeger.yaml (hunk 1 above)
Comment (phlax) [reply examples/jaeger-tracing/front-envoy-jaeger.yaml:20]: ah, k - i wasnt sure - ill update

**Review by htuch — APPROVED** [review]
LGTM, thanks!

**Comment by phlax** [comment]
not sure what is happening with CI - this PR was passing with current config earlier

now im seeing errors about `xstatic_resources` across failing PRs

**Comment by phlax** [comment]
sorry my bad (emacs key bindings) - ill fix

**Review by mattklein123 — APPROVED** [review]

**Comment by moderation** [comment]
Woohoo @phlax. More v2 if you have the bandwidth. Here are the files with v2 in the name and there are more v2 references inside `configs/`:
```
configs/access_log_format_helper_v2.template.yaml
configs/envoy_double_proxy_v2.template.yaml
configs/envoy_front_proxy_v2.template.yaml
configs/envoy_router_v2.template.yaml
configs/envoy_service_to_service_v2.template.yaml
configs/google_com_proxy.v2.yaml
configs/routing_helper_v2.template.yaml
configs/using_deprecated_config.v2.yaml
examples/ext_authz/config/grpc-service/v2.yaml
examples/ext_authz/config/opa-service/v2.yaml
test/config/integration/google_com_proxy_port_0.v2.yaml
```

**Pull request merged**